photo_points/Views/Admin/SearchPhotoPoints.cshtml

```diff
@@ -0,0 +1,64 @@
+@*@model photo_points.Controllers.AdminController*@
+@model photo_points.ViewModels.SearchViewModels
+
+@addTagHelper *,Microsoft.AspNetCore.Mvc.TagHelpers
+@{
+    Layout = "_Layout";
+    ViewBag.Title = "Search";
+}
+
+<h3>Search Photos</h3>
+
+<table>
```

Comment:
Search options are good. I might add whether the photo is approved or rejected, but that is not as important as what you already have in the weight of the search.

Reply:
good point about the approval type . I'll add the option to the UI.